recipes/muparser/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+sh ./configure --prefix=$PREFIX --disable-samples
+make -j2
```

Comment:
I have never seen use of -j in any Linux packaging scripts. Maybe they have
a reason?

On Wed, May 4, 2016, 5:06 PM jakirkham <notifications@github.com> wrote:

> In recipes/muparser/build.sh
> <https://github.com/conda-forge/staged-recipes/pull/530#discussion_r62056128>
> :
>
> > @@ -0,0 +1,5 @@
> > +#!/bin/sh
> > +
> > +sh ./configure --prefix=$PREFIX --disable-samples
> > +make -j2
>
> I had the impression that Travis CI had a limit of 1.5 (kind of weird but
> that's what it was). Though I believe I saw that for Linux so I don't
> remember what they do on Mac.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/conda-forge/staged-recipes/pull/530/files/b601f69d72d22eec76744804d1d539e293148b87#r62056128>
>